db/helpers/utils.js

```diff
@@ -1,3 +1,6 @@
+const { readFile } = require('fs');
+const fs = require('fs/promises');
```

Comment:
You'll need one of these imports but not both. You're requiring both `fs` and `fs/promises`, when you only need one.